Add Atmos Design Patterns docs (#486)
cloudposse/atmos: website/docs/design-patterns/summary.md

---
title: Summary
sidebar_position: 100
sidebar_label: Summary
description: Summary
---

Architecting and provisioning enterprise-grade infrastructure is challenging, and making all the components and
configurations [reusable](https://en.wikipedia.org/wiki/Reusability) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is even more
difficult.

> Atmos Design Patterns can help with structuring and organizing your design to provision multi-account enterprise-grade environments for complex

Reviewer: ```suggestion
These Atmos Design Patterns can help with structuring and organizing your design to provision multi-account enterprise-grade environments for complex
```

organizations.

We've already identified and documented many Atmos Design Patterns.

> You might be asking yourself: "How do I start with all of that and what steps to take?"

Reviewer: ```suggestion
You may be wondering, "Where do I begin and what steps should I follow?"
```

Here are some recommendations:

- This [Quick Start](https://github.com/cloudposse/atmos/tree/master/examples/quick-start) repository presents an example of an infrastructure managed
  by Atmos. You can clone it and configure to your own needs. The repository should be a good start to get yourself familiar with Atmos and the
  Design Patterns. The [Quick Start Guide](/category/quick-start) describes the step required to configure and start using the repository

> - If you are just developing or testing your infrastructure, you can use any of the Design Patterns described in this guide, and find out which ones
>   are best suited to your requirements

Reviewer: ```suggestion
- If you are just developing or kicking the tires on Atmos, you can use any of the Design Patterns described in this guide, and find out which ones
  are best suited to your requirements. The easiest ones to get started with will be:
   - [Inline Component Configuration](/design-patterns/inline-component-configuration)
   - [Inline Component Customization](/design-patterns/inline-component-customization)
```
